posthog/temporal/data_imports/pipelines/sql_database/__init__.py

```diff
@@ -195,13 +259,18 @@ def get_column_hints(engine: Engine, schema_name: str, table_name: str) -> dict[
     columns: dict[str, TColumnSchema] = {}
 
     for column_name, data_type, numeric_precision, numeric_scale in results:
-        if data_type != "numeric":
-            continue
-
-        columns[column_name] = {
-            "data_type": "decimal",
-            "precision": numeric_precision or 76,
-            "scale": numeric_scale or 16,
-        }
+        if data_type == "numeric":
+            columns[column_name] = {
+                "data_type": "decimal",
+                "precision": numeric_precision or 76,
+                "scale": numeric_scale or 16,
+                "nullable": True,
+            }
+        else:
+            columns[column_name] = {
+                "name": column_name,
+                "data_type": POSTGRES_TO_DLT_TYPES.get(data_type, "text"),
+                "nullable": True,
```

Comment:
I didn't do this in my original PR, but we should read `is_nullable` from `information_schema.columns` too